web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -147,14 +147,13 @@
   let shiftKeyIsDown = false;
 
   const onKeyDown = (e: KeyboardEvent) => {
-    if (e.shiftKey && e.key !== '/') {
+    if (e.key === 'Shift') {
```

Comment:
The issue is the e.preventDefault. Why do we need to prevent the default behavior for tracking if the shift key is up or down?

Reply:
'/' is a shortcut in firefox which enables quick find

Reply:
Moreover, I have a french keyboard layout so to trigger `/` I need to press `Shift` and `:`

Reply:
Ah, we should consider adding a comment for that specific combination as we do indeed want to prevent the default action for that.